membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -40,18 +42,18 @@ class AccountController extends LazyLogging {
 
     // TODO - refactor to use the Zuora-only based lookup, like in AttributeController.pickAttributes - https://trello.com/c/RlESb8jG
 
-    val updateForm = Form { single("stripeToken" -> nonEmptyText) }
+    val updateForm = Form { tuple("stripeToken" -> nonEmptyText, "publicKey" -> text) }
     val tp = request.touchpoint
     val maybeUserId = authenticationService.userId
     logger.info(s"Attempting to update card for $maybeUserId")
     (for {
-      user <- EitherT(Future.successful( maybeUserId \/> "no identity cookie for user"))
-      stripeCardToken <- EitherT(Future.successful(updateForm.bindFromRequest().value \/> "no card token submitted with request"))
+      user <- EitherT(Future.successful(maybeUserId \/> "no identity cookie for user"))
+      stripeDetails <- EitherT(Future.successful(updateForm.bindFromRequest().value \/> "no card token submitted with request"))
+      (stripeCardToken, stripePublicKey) = stripeDetails
```

Comment:
could this have been done on the line before? it doesn't look like we use the `stripeDetails` val anywhere as it is

Reply:
Maybe you can help me with this? https://stackoverflow.com/questions/32651337/scala-for-comprehension-with-tuple-decomposition